db/models.py

```diff
@@ -1,4 +1,11 @@
 from django.db import models
+from django.db.models import UniqueConstraint
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+
+
+class User(AbstractUser):
+    pass
```

Comment:
The `User` model extends `AbstractUser`. Ensure that any additional fields or methods needed for your application are added here.